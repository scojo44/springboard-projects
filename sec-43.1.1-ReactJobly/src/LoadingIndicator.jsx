import React from 'react'
import './LoadingIndicator.css'

export default function LoadingIndicator() {
  return (
    <section className="LoadingIndicator">
      Loading...
    </section>
  );
}
