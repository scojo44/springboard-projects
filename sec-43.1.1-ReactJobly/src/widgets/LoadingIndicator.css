.LoadingIndicator {
  flex: 1;
  text-align: center;
  font-size: 10vw;
  margin-top: 10vh;
}