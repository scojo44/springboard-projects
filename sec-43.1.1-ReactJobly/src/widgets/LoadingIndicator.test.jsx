import {render} from '@testing-library/react'
import {MemoryRouter} from 'react-router-dom'
import LoadingIndicator from './LoadingIndicator'

describe('LoadingIndicator Tests', () => {
  it('Renders without crashing', () => {
    render(
      <MemoryRouter>
        <LoadingIndicator />
      </MemoryRouter>
    );
  });

  it('Matches snapshot', () => {
    const {asFragment} = render(
      <MemoryRouter>
        <LoadingIndicator />
      </MemoryRouter>
    );

    expect(asFragment()).toMatchSnapshot();
  });
});
