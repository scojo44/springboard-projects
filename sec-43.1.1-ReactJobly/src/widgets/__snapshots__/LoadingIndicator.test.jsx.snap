// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`LoadingIndicator Tests > Matches snapshot 1`] = `
<DocumentFragment>
  <section
    class="LoadingIndicator"
  >
    Loading...
  </section>
</DocumentFragment>
`;
